Sources/MapboxMaps/Foundation/MapboxMap.swift

```diff
@@ -835,6 +841,59 @@ extension MapboxMap {
                                  featureId: featureId,
                                  stateKey: stateKey)
     }
+
+}
+
+// MARK: - View Annotations
+
+extension MapboxMap {
+
+    // TODO: Add documentation
+    public func setViewAnnotationPositionsUpdateListenerFor(listener: ViewAnnotationPositionsListener) {
```

Comment:
The name suggests that there can only be 1 listener at a time. Is that true?

Reply:
Yes, and it's one more reason to make this function internal. 👌